server/chat-plugins/helptickets.js

```diff
@@ -69,6 +70,21 @@ function writeTickets() {
 	));
 }
 
+/**
+ * @param {string} line
+ */
+function writeStats(line) {
+	// ticketType,totalTime,timeToFirstClaim,inactiveTime,resolution,result,staff;userids;seperated;with;semicolons
+	const date = new Date();
+	const month = Chat.toTimestamp(date).split(' ')[0].split('-', 2).join('-');
+	try {
+		// TODO dont start with \n if file is empty
```

Comment:
Note to self.

Worst case, I can just have the parser handle a blank line at the end of the file.